src/core/ee/dmac.cpp

```diff
@@ -140,6 +140,43 @@ void DMAC::run(int cycles)
     }
 }
 
+bool DMAC::mfifo_handler(int index)
```

Comment:
Perhaps add a short comment explaining what MFIFO does and when this function returns true/false?